RESULTS.md

```diff
@@ -0,0 +1,91 @@
+ 
+### BabylonJS
+1. Due to Security policies in Chrome, the AudioContext is not allowed to start until user input/interaction is done with the webpage. This meant that the prototype could not speak an initial welcome message, and was remedied by manually unlocking the AudioContext engine before the welcome message was played.
+2. GUI Text Input does not function properly on a mobile. This is a currently known issue within the BabylonJS documentation, the Text Input element cannot invoke the onscreen keyboard so as a workaround it sends an alert window to get the input. This however causes the AR world to freeze as on mobile this prompt window does not get sent to the front above the AR world. 
+4. Unable to take a screenshot of the AR world including the video stream. In an attempt to satisfy the requirement in Table 1 to verify documents in order to validate a user, there was an attempt to use the internal BabylonJS tools to take a screenshot through a makeshift viewfinder. This failed as detailed in a GitHub issue where the screenshot tools could only access and take a screenshot of the AR world but not the underlying video stream.
+
+### Azure Cognitive Services
+1. The neural trained voices provided in the Azure Cognitive services still sound robotic and unnatural. There is no workaround for this. 
+2. Input only accepted in WAV file format. This was an issue since most browser MediaRecorders do not export to the WAVE/WAV file codecs, the solution to this was to use the "audiobuffer-to-wav" library in order to insert the codec headers into the file to make it compatible with the Azure Cognitive Services file upload process. 
+
+## Considerations
+### Current Limitations
+Since the WebXR standard at the moment is a work in progress, it is expected that the functionality and widespread use of this technology will be possible when version 1 is released. At this time the technology and its adoption is too limited to make a widespread application as the full features are limited to Android OS and the Chrome browser.
+
+### Possible Improvements to Prototype
+Quality of life improvements could be made to the prototype in its current state such as performance increases on a response from the server. This would allow for the time spent processing a request to be diminished but not by much since most of the delay in introduced waiting for a response from the cloud services rather than the API and WebXR itself. 
+
+Another improvement could be the addition of a skip button, which would allow users to skip the response if they know that the response is not what they are looking for, similar to interrupting an agent in a service centre to clarify a question. This would avoid a user being stuck listening to a response that they know is irrelevant for the entirity of the text to speech response.
+
+## References
+TODO: Link the issues as references and the babylonjs documentation.
```

Comment:
If references available add, otherwise drop

Reply:
Put in direct links to resources instead of a references section.